mlflow/pyfunc/model.py

```diff
@@ -583,7 +610,14 @@ def _convert_input(self, model_input):
             elif isinstance(model_input, list) and all(isinstance(x, dict) for x in model_input):
                 keys = [x.name for x in self.signature.inputs]
                 return [{k: d[k] for k in keys} for d in model_input]
-
+        elif isinstance(hints.input, type) and (
+            issubclass(hints.input, ChatCompletionRequest)
```

Comment:
The naming of `ChatCompletionRequest` is a bit confusing, since it doesn't match the OpenAI chat completions format. Can we rename this to something like `SimpleChatCompletionRequest`?

Reply:
I have an open thread with @epec254 about the naming of the rag_signature objects. Will pass this feedback along.